### DISCUSSION TITLE
OpenAI 首席科學家：AI 可能開始擁有意識

### DISCUSSION THREAD

**Discussion by Yao900108** [1101-GS4719-C/Python-Community-GS4719-C#529] [科技新知]
OpenAI 首席科學家 Ilya Sutskever 最近在 Twitter 表示：「目前大型神經網路可能稍微擁有自我意識。」他在人工智慧範疇有多年經驗，曾受訪時表示通用人工智慧系統雖然有廣泛應用，不過卻可能有獨裁統治的影響力，須密切注意發展。
通用人工智慧（AGI）指的是可像一般人思考，無需額外編寫程式就能配合各種情況學習，自行解決新問題，有更大應用潛力。相對 AGI，現在常見的人工智慧都只是「弱人工智慧」，只能應付特定用途，無法通用。不過 Ilya Sutskever 未進一步解釋，到底是 OpenAI 有新進展還是他察覺到其他人工智慧有變化。
以前常看到以關於AI產生自我意識而背叛人類之類的主題為題材拍攝的電影，也許未來真的會發生。我們必須更加慎重看待此事。
https://technews.tw/2022/02/16/openai-large-neural-networks-are-slightly-conscious/

**Comment by sean1325418** [comment]
AI學習東西的速度比人類快太多了，但也不用太擔心

**Comment by 11timyyy** [comment]
擁有自我意識的話真的很可怕....

**Comment by ghost** [comment]
但如果這是科技終末勢必演變的結果，我覺得往往過程中多麼防範，始終都會發生

**Comment by Sazmafuji** [comment]
希望此往好處發展，看看未來會發生甚麼事

**Comment by ThomasMeng0515** [comment]
之前聽教授說，AI的發展在軟體上進步得很快，但是硬體上似乎還沒有跟上軟體發展的腳步，所以我覺得在硬體沒有能力配合的情況下，要產生有意識的AI感覺很困難

**Comment by PoooYen** [comment]
要怎麼明確定義自我意識好像有點複雜

**Comment by HuanYuuuuuu** [comment]
AI學習速度的確非常快速，但要如何去定義擁有意識也是一個問題

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，期望AI不要造成太大的困難。

**Comment by cordydeng** [comment]
如果未來AI真的有自我意識，不知道人們會不會被淘汰，希望未來AI的角色還是輔助人們的工具，而不是我們的主人

**Comment by awkward-willy** [comment]
就我而言，意識是察覺、知道自己的存在，難以想像電腦是如何表達出這回事的，Ilya 的內容也難以察覺出端倪，蠻好奇是否有誇大的成分
